caluma/caluma_core/events.py

```diff
@@ -1,18 +1,26 @@
 import logging
 from warnings import warn
 
-from django.dispatch import receiver
-
 logger = logging.getLogger(__name__)
 
 
-def on(event, *args, **kwargs):
+def on(event, raise_exceptions=False, *args, **kwargs):
     deprecation_reason = getattr(event, "_deprecation_reason", None)
 
     if deprecation_reason:  # pragma: no cover
         warn(deprecation_reason, DeprecationWarning)
 
-    return receiver(event, *args, **kwargs)
+    def _decorator(func):
+        func._raise_exceptions = raise_exceptions
+
+        if isinstance(event, (list, tuple)):
```

Comment:
Not sure this is a good idea. I'd prefer chaining decorators over this

Reply:
This is copied from the original implementation of djangos receiver. Removing it would be a breaking change.

Reply:
Fair point, in this case we *must* keep it :)